pkg/console/subresource/configmap/configmap.go

```diff
@@ -51,16 +49,26 @@ func DefaultConfigMap(
 		Brand(DEFAULT_BRAND).
 		DocURL(DEFAULT_DOC_URL).
 		APIServerURL(getApiUrl(infrastructureConfig)).
+		CustomLogoFile(defaultCustomLogoFile).
+		CustomProductName(defaultCustomProductName).
 		ConfigYAML()
 
 	extractedManagedConfig := extractYAML(managedConfig)
 
+	customProductName := operatorConfig.Spec.Customization.CustomProductName
+	customLogoFile := operatorConfig.Spec.Customization.CustomLogoFile.Key
+	if customLogoFile != "" {
```

Comment:
Push this little prefixer of `/var/logo` up into the `Builder. CustomLogoFile()` func?  We should be able to stay tidy here now.

Reply:
excellent idea